### PR TITLE
Improve markup for resistor colour codes

### PR DESCRIPTION
**Pull request by roshanr95**
This improves the markup for the resistor colour codes goodie as per #851.

screen shot 2015-01-15 at 2 56 42 am

Is there a standard `zci__something` class/style for the link at the bottom?

### CODE REVIEW

**Comment by jagtalon** [comment]
@roshanr95 thanks! We'll get back to you soon. :)

**Comment by mintsoft** [comment]
@roshanr95 Thanks for this; it LGTM! I like the move from inline-styling to named classes.

I just want to hook in the design guys incase there are any colour faux-pas here.

@chrismorast or @abeyang do you have any suggestions for the shades of the colours used here: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/921/files#diff-29590398a8a6e3105ce830e4091829a8R8

**Comment by chrismorast** [comment]
Nice! Thanks @roshanr95 , this is looking great - I'm really stoked to see this one get done!

@jagtalon , I will put together a revised list of colors and post tomorrow.

Also, the attribution should be color # 888 and preceded by a favicon (or placeholder favicon if there isn't one available). @jagtalon, can you point @roshanr95 to that?

**Comment by chrismorast** [comment]
Here are the new colors:
Silver  #d0d0d0;
gold #f2d374;
black #333333;
brown #726258;
red #de4f33;
orange #f4912d;
yellow #ffc240;
green #8fc547;
blue #4a96d1;
purple #8461a7;
gray #a0a0a0;

Visual reference:
screen shot 2015-01-16 at 3 52 26 pm

File: lib/DDG/Goodie/ResistorColors.pm [hunk 1]
```diff
@@ -158,17 +164,18 @@ sub render {
...
+            $html .= "<span class='resistor-band $class'>$name $html_prefix$display_digit</span>";
         } else {
             return;
         }
         $index++;
     }
-    $html .= "<br/>"
+    $html .= "</div></div>"
+        . "<br/>"
         . "<a href='http://resisto.rs/#$formatted_value' class='resistorlink'>"
```
Comment (mintsoft) [inline lib/DDG/Goodie/ResistorColors.pm]: I think the resistorlink class isn't needed here.

**Comment by roshanr95** [comment]
Changed the colours and the link style. What favicon do I use?

**Comment by mintsoft** [comment]
@roshanr95 Do you mean the favicon for the link?

**Comment by roshanr95** [comment]
@mintsoft Yeah, for the resisto.rs link. That was what @chrismorast meant by attribution right?

**Comment by mintsoft** [comment]
@roshanr95 yes exactly; it doesn't look there's a favicon for `resisto.rs` so the usual practice is to just omit it.

**Comment by roshanr95** [comment]
Oh ok..

**Comment by mintsoft** [comment]
Terrific! Thanks for the work @roshanr95 !

**Comment by jagtalon** [comment]
@roshanr95 @mintsoft @chrismorast 

terrrific

**Comment by jagtalon** [comment]
@roshanr95 you might be interested in https://github.com/duckduckgo/zeroclickinfo-goodies/issues/934

**Comment by chrismorast** [comment]
Nice work @roshanr95 ! 
One thing I notice, it looks like we're doubling up on the left padding...

screen shot 2015-01-19 at 9 30 57 pm
screen shot 2015-01-19 at 9 30 50 pm

Can we remove it from one of those divs so it lines up with the left rail of the rest of the page's content like this...
screen shot 2015-01-19 at 9 30 36 pm

Thanks again for all your work on this!

**Comment by mintsoft** [comment]
@chrismorast sorry! Missed that completely

**Comment by mintsoft** [comment]
@roshanr95 If you would like to address what @chrismorast says about the padding you'll unfortunately have to open another PR. Muggins here has merged this way too soon! Ping me if you don't want to make the changes and I'll tidy it

**Comment by jagtalon** [comment]
@roshanr95 @mintsoft @chrismorast I'll open an issue for it!

@mintsoft it's fine haha! It did look like it was good enough.

**Pull request merged**